include/grpc/impl/channel_arg_names.h

```diff
@@ -223,20 +234,22 @@
 /** The maximum amount of memory used by trace events per channel trace node.
  * Once the maximum is reached, subsequent events will evict the oldest events
  * from the buffer. The unit for this knob is bytes. Setting it to zero causes
- * channel tracing to be disabled. */
+ * channel tracing to be disabled. Defaults to (1024 * 4) bytes. */
 #define GRPC_ARG_MAX_CHANNEL_TRACE_EVENT_MEMORY_PER_NODE \
   "grpc.max_channel_trace_event_memory_per_node"
 /** If non-zero, gRPC library will track stats and information at at per channel
  * level. Disabling channelz naturally disables channel tracing. The default
  * is for channelz to be enabled. */
 #define GRPC_ARG_ENABLE_CHANNELZ "grpc.enable_channelz"
 /** Channel arg (integer) setting how large a slice to try and read from the
-   wire each time recvmsg (or equivalent) is called **/
+   wire each time recvmsg (or equivalent) is called Range varied from 8192
+   to (32 * 1024 * 1024) bytes. **/
 #define GRPC_ARG_TCP_READ_CHUNK_SIZE "grpc.experimental.tcp_read_chunk_size"
```

Comment:
here, the default seems to be 8192, and the range of accepted values is 1 to 32*1024*1024